charts/airbyte/values.yaml

```diff
@@ -0,0 +1,380 @@
+  ## @param webapp.image.pullPolicy the pull policy to use for the airbyte webapp image
+  ## @param webapp.image.tag The airbyte webapp image tag. Defaults to the chart's AppVersion
+  image:
+    repository: airbyte/webapp
+    pullPolicy: IfNotPresent
+    tag: 0.29.13-alpha
+
+  ## @param webapp.podAnnotations [object] Add extra annotations to the scheduler pod
+  ##
+  podAnnotations: {}
+
+  ## @param webapp.service.type The service type to use for the webapp service
+  ## @param webapp.service.port The service port to expose the webapp on
+  service:
+    type: ClusterIP
+    port: 80
+
+  ## Web app resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param webapp.resources.limits [object] The resources limits for the Web container
+  ## @param webapp.resources.requests [object] The requested resources for the Web container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 200m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    memory: 256Mi
+    ##    cpu: 250m
+    requests: {}
+
+  ## @param webapp.nodeSelector [object] Node labels for pod assignment
+  ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+  ##
+  nodeSelector: {}
+
+  ## @param webapp.tolerations [array] Tolerations for webapp pod assignment.
+  ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  ##
+  tolerations: []
+
+  ## Configure the ingress resource that allows you to access the Airbyte installation.
+  ## ref: http://kubernetes.io/docs/user-guide/ingress/
+  ## @param webapp.ingress.enabled Set to true to enable ingress record generation
+  ## @param webapp.ingress.className Specifies ingressClassName for clusters >= 1.18+
+  ## @param webapp.ingress.hosts Ingress Hosts configuration
+  ## @param webapp.ingress.annotations [object] Ingress annotations done as key:value pairs
+  ## @param webapp.ingress.hosts The list of hostnames to be covered with this ingress record.
+  ## @param webapp.ingress.tls [array] Custom ingress TLS configuration
+  ingress:
+    enabled: false
+    className: ""
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    hosts:
+    - host: chart-example.local
+      paths:
+      - path: /
+        pathType: ImplementationSpecific
+    tls: []
+    # - secretName: chart-example-tls
+    #   hosts:
+    #   - chart-example.local
+
+
+## @section Scheduler Parameters
+
+scheduler:
+  ## @param scheduler.replicaCount Number of scheduler replicas
+  replicaCount: 1
```

Comment:
This isn't safe to have >1 at the moment. In the future we'll add support for scaling up schedulers. We could either remove the configurability or just add a comment saying this should always be 1.